python/nano/test/run-nano-pytorch-ray-tests.sh

```diff
@@ -11,7 +11,10 @@ set -e
 echo "# Start testing"
 start=$(date "+%s")
 python -m pytest -s ${NANO_RAY_TEST_DIR}/test_ray_trainer.py ${NANO_RAY_TEST_DIR}/test_torch_nano_ray.py
-python -m pytest -s ${ANALYTICS_ZOO_ROOT}/python/nano/test/pytorch/tests/train/trainer/test_scale_lr.py -k 'ray'
+version=$(python -c "import torch;print(torch.__version__)")
+if [ ${version} != "2.0.0+cu117" ];then  # workaround as we have not upgrade pl
```

Comment:
should we use sth similar to `startwith`?

Reply:
Done